src/python/bot/tasks/fuzz_task.py

```diff
@@ -1453,7 +1453,10 @@ def generate_blackbox_testcases(self, fuzzer, fuzzer_directory,
         return error_occurred, None, None, None, None
 
       # Build the fuzzer command execution string.
-      command = shell.get_execute_command(fuzzer_executable)
+      # TODO(mbarbella): Conditionally invoke with Python 2 depending on
+      # the fuzzer. At present, all were implemented to work with Python 2.
+      command = shell.get_execute_command(
```

Comment:
Can you make this rely on an env variable
USE_PYTHON2_FOR_BLACKBOX_FUZZERS
and then set that env in project.yaml for chromium.
I don't want this to happen for other CF users.

Reply:
Good call. Done.